### PR TITLE
REF: SquareButton to TS

### PR DESCRIPTION
**Pull request by Danswar**
@limpbrains Would you like to take a look?

Based on the usage of this component, I removed some of the props that are no longer in use, the same goes for code bound to those props and always-overwritten styles.

### CODE REVIEW

**Review by Overtorment — APPROVED** [review]
lgtm

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Review by limpbrains — CHANGES_REQUESTED** [review]

File: components/SquareButton.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  const hookStyles = StyleSheet.create({
+    contentContainer: {
+      flexDirection: 'row',
+      justifyContent: 'center',
+      alignItems: 'center',
+    },
+    text: {
+      marginHorizontal: 8,
+      fontSize: 16,
```
[the remark this answers is not in the document]
Comment (Danswar) [reply components/SquareButton.tsx]: solved

File: components/SquareButton.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+/* eslint react/prop-types: "off", react-native/no-inline-styles: "off" */
```
[the remark this answers is not in the document]
Comment (Danswar) [reply components/SquareButton.tsx]: done

File: components/SquareButton.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+interface SquareButtonProps {
+  title: string;
+  onPress: () => void;
+  style: StyleProp<ViewStyle>;
+  testID?: string;
+}
+
+export const SquareButton = forwardRef<TouchableOpacity, SquareButtonProps>((props, ref) => {
+  const { title, onPress, style, testID = '' } = props;
```
[the remark this answers is not in the document]
Comment (Danswar) [reply components/SquareButton.tsx]: OMG, done thanks!

**Comment by Danswar** [comment]
@limpbrains fancy to have a second look? thanks for catching those details.

**Review by limpbrains — APPROVED** [review]
awesome work

**Pull request merged**